atom/browser/api/lib/power-save-blocker.coffee

```diff
@@ -0,0 +1,8 @@
+bindings = process.atomBinding 'power_save_blocker'
+
+powerSaveBlocker = bindings.powerSaveBlocker
+
+powerSaveBlocker.PREVENT_APP_SUSPENSION = 0
+powerSaveBlocker.PREVENT_DISPLAY_SLEEP = 1
```

Comment:
We probablely cannot do it here. The above scripts will translate to 
```
powerSaveBlocker = {
  PREVENT_APP_SUSPENSION: 0,
  PREVENT_DISPLAY_SLEEP: 1
};
```
which only exposes two attributes.